application/test_data.csv

```diff
@@ -0,0 +1,11 @@
+,track_id,title,song_id,release,artist_id,artist_mbid,artist_name,duration,artist_familiarity,artist_hotttnesss,year,genre,lyrics
```

Comment:
Should we add this file to .gitignore? This feels like it might be the one exception to that where we might wanna keep it in, provided it stays small.

Reply:
Let's keep it just for now, we can remove it from the release version